README.md

```diff
@@ -9,15 +9,23 @@
 
 ---
 
-Our templates offer a rich boilerplate to jump start React-based application development for [Create React App](https://github.com/facebook/create-react-app).
+<p align="center">
+    <a href="https://www.npmjs.com/package/cra-template-nimble"><img src="https://badgen.net/npm/v/cra-template-nimble" /></a>
+    <a href="https://www.npmjs.com/package/cra-template-nimble"><img src="https://badgen.net/npm/dy/cra-template-nimble" /></a>
+</p>
+
+Our templates offer a rich boilerplate to jump-start React-based application development for [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
Not sure but should we remove the `s` here as this is only one template 🤔 

```suggestion
Our template offers a rich boilerplate to jump-start React-based application development for [Create React App](https://github.com/facebook/create-react-app).
```

Reply:
✅ c063f6f ;-)